forest/daemon/src/daemon.rs

```diff
@@ -398,18 +391,51 @@ pub(super) async fn start(config: Config, detached: bool) {
     info!("Forest finish shutdown");
 }
 
+async fn prompt_and_fetch_snapshot(config: &mut Config) {
+    if !config.client.download_snapshot {
+        if let false = Confirm::with_theme(&ColorfulTheme::default())
```

Comment:
isn't `if let false = fn()` just `if !fn()`?

Reply:
Fixed and change to a match for error handling over expect